src/plugins/links/public/components/editor/links_editor.tsx

```diff
@@ -172,7 +175,12 @@ const LinksEditor = ({
           <EuiFlexItem grow={false}>
             <EuiToolTip content={LinksStrings.editor.panelEditor.getTechnicalPreviewTooltip()}>
               {/* The EuiBadge needs an empty title to prevent the default tooltip */}
-              <EuiBadge color="hollow" tabIndex={0} title="">
+              <EuiBadge
+                color="hollow"
+                tabIndex={0}
+                title=""
+                aria-label={LinksStrings.editor.panelEditor.getTechnicalPreviewTooltip()}
+              >
                 {LinksStrings.editor.panelEditor.getTechnicalPreviewLabel()}
               </EuiBadge>
             </EuiToolTip>
```

Comment:
Maybe the [EuiBetaBadge](https://eui.elastic.co/#/display/badge#beta-badge-type) is a better and more consistent alternative. I only discovered EuiBetaBadge today. 🧑‍🏫 

```suggestion
              <EuiBetaBadge
                color="hollow"
                size="s"
                label={LinksStrings.editor.panelEditor.getTechnicalPreviewLabel()}
                aria-label={LinksStrings.editor.panelEditor.getTechnicalPreviewTooltip()}
              />
```

Reply:
I actually looked around when initially adding the `Technical preview` badge, and the default `EuiBadge` was much more commonly used for "Technical preview" labels - that's why I went with it originally. Since it will only be around for `8.13` and removed in `8.14` when we GA, I wonder if it's worth changing at this stage? :eyes:

Reply:
Probably not worth changing this then. 👍